apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineManager.java

```diff
@@ -447,9 +447,8 @@ public PipelineJobOutcome call() throws Exception {
         throw new InterruptedException("Re-firing job interrupt.");
       } catch (Exception e) {
         jobRecordStore.recordJobFailure(jobRecord.getId(), new PipelineJobFailure(e));
-        LOGGER.warn(String.format("Job failed: jobRecord='%s'", jobRecord), e);
 
```

Comment:
What's the advantage of removing the log message here?

Reply:
Given the warning added over in `PipelineJobRecordStore`, it feels redundant to me. Disagree?

Reply:
Yep.  I missed that before.  Looks good.